centreon/src/Core/Media/Infrastructure/API/AddMedia/AddMediaController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types = 1);
+
+namespace Core\Media\Infrastructure\API\AddMedia;
+
+use Centreon\Application\Controller\AbstractController;
+use Centreon\Domain\Log\LoggerTrait;
+use Core\Application\Common\UseCase\ErrorResponse;
+use Core\Common\Infrastructure\Upload\FileIterator;
+use Core\Media\Application\UseCase\AddMedia\AddMedia;
+use Core\Media\Application\UseCase\AddMedia\AddMediaRequest;
+use Core\Media\Infrastructure\API\Exception\AddMediaException;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class AddMediaController extends AbstractController
```

Comment:
phpstan not passed : must be final etc ...
```suggestion
final class AddMediaController extends AbstractController
```